src/applications/caregivers/config/chapters/primary/identyInformation.js

```diff
@@ -0,0 +1,24 @@
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
+import { ssnUI } from '../../../definitions/sharedUI';
+import CaregiverSsnDescription from '../../../components/FormDescriptions/CaregiverSsnDescription';
+import content from '../../../locales/en/content.json';
+import fullSchema from '../../10-10CG-schema.json';
+
+const { ssn } = fullSchema.definitions;
+const inputLabel = content['primary-input-label'];
+
+const primaryIdentityInformation = {
+  uiSchema: {
+    ...titleUI(content['primary-info-title--id'], CaregiverSsnDescription),
+    primarySsnOrTin: ssnUI({ label: inputLabel, type: 'ssntin' }),
```

Comment:
The `type` on the ssn/tin UI schema is simply a way to control the content displayed. We have one instance (for the Veteran) where the stakeholder only wants to reference SSN, whereas with the caregivers, we can reference SSN or TIN.